pilots/10_things_to_know_about_pilots.Rmd

```diff
@@ -0,0 +1,189 @@
+
+This does not entail that one needs to show that effect sizes are large enough nor that hypotheses are likely to be confirmed. Instead, a pilot can demonstrate that your research project is feasible in terms of time and resources, that your study design is adequate for answering the research questions proposed, and that your research team has the expertise and capacity to administer the study, perform analyses, and even present results in a compelling fashion [(van Teijlingen et al. 2001)](https://doi.org/10.1046/j.1365-2648.2001.01757.x). In a similar fashion, piloting can help you to recruit study team members and organizational partners, or solicit institutional support.
+
+# 6. You can use a pilot to get feedback on your study design
+
+Don't keep those pilot results to yourself! Sharing initial findings, successes, and challenges is a great way to help you prepare for your main study. In light of the growth of the open science movement [(Christensen et al. 2020)](https://escholarship.org/uc/item/0hx0207r), conferences and workshops are increasingly open to accepting submissions based on pre-analysis plans and pilot results. Whether through these more formal venues, or by reaching out to colleagues or experts, you can use pilot results to receive feedback about your study design, such as strategies to address possible challenges and unexplored theoretical or empirical directions that you can incorporate in your main study. 
+
+Further, it can take a long time to complete an experimental study and publish results. Sharing intermediate findings allows you to coordinate with other researchers in the field, helping you to align your work and incorporate recent theoretical and empirical innovations relevant to your study.
+
+Ultimately, the design and piloting stage is the **best** time to receive feedback, as you still have time to make improvements. In contrast, key research design decisions may be locked into place by the time your main study results and formal peer review come around.
+
+# 7. Keep an eye out for ethical considerations when piloting
+
+When you design your study initially, all of the relevant ethical considerations and risks may not be immediately apparent. The piloting stage is thus a good opportunity to review whether any risks or harms you anticipated may come into play and whether still other ethical considerations should be incorporated into your main study.

```

Comment:
link to EGAP ethics statement: https://egap.org/wp-content/uploads/2020/05/egap-committee-memo-on-the-report-of-the-apsa-ad-hoc-committee-on-human-subjects-research.pdf